### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="contributing\.html"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14480]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
li:nth-child(4) > a[href$="contributing\.html"]
#### Snippet
`<a href="/contributing.html">Contributing</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.3 (foreground color: #ffffff, background color: #43a047, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
